conda/recipes/dask-cudf/meta.yaml

```diff
@@ -13,12 +13,12 @@ build:
 requirements:
   build:
     - python x.x
-    - cudf 0.6*
+    - cudf 0.7*
     - dask >=0.19.0
     - distributed >=1.23.0
   run:
     - python x.x
-    - cudf 0.6*
+    - cudf 0.7*
```

Comment:
Do we need to pin this so tightly? It sounds like that is what is causing us issues.

Reply:
Yes, because the cudf API is rapidly evolving and breaking with every release.

Reply:
Makes sense. Thanks for clarifying.